### PR TITLE
DOC-11608: Remove example tags from Atomic

### PR DESCRIPTION
**Pull request by jpmarceau**
The way our docs generation goes at the moment, we should put the atomic prop examples in the descriptions rather than in example tags. 
I missed a few when reviewing PRs, so this PR here is to correct that 🙂

### CODE REVIEW

File: packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx [hunk 1]
```diff
@@ -158,7 +158,7 @@ export class AtomicSearchBox {
   /**
    * Event that is emitted when a standalone search box redirection is triggered. By default, the search box will directly change the URL and redirect accordingly, so if you want to handle the redirection differently, use this event.
```
Comment (jpmarceau) [inline packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx:159]: Actually, I notice that we don't generate docs for events currently. I'll add that to the list :loading:

File: packages/atomic/src/components/search/result-template-components/result-template-decorators.tsx [hunk 2]
```diff
@@ -20,8 +20,7 @@ export class MissingResultParentError extends Error {
  * A [StencilJS property decorator](https://stenciljs.com/) to be used for result template components.
  * This allows the Stencil component to fetch the current result from its rendered parent, the `atomic-result` component.
  *
- *
- * @example
+ * Example:
```
Comment (jpmarceau) [inline packages/atomic/src/components/search/result-template-components/result-template-decorators.tsx:23]: We don't generate docs for this anyway, but I thought I'd take care of it in this PR just for uniformity 🙂

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic/src/components/search/result-template-components/atomic-result-link/atomic-result-link.tsx [hunk 3]
```diff
@@ -55,8 +55,7 @@ export class AtomicResultLink implements InitializableComponent {
    *
    * The template literal can reference any number of result properties from the parent result. It can also reference the window object.
    *
-   * @example
-   * The following markup generates an `href` value such as `http://uri.com?id=itemTitle`:
+   * For example, the following markup generates an `href` value such as `http://uri.com?id=itemTitle`.
    * <atomic-result-link href-template='${clickUri}?id=${raw.itemtitle}'></atomic-result-link>
```
Comment (jpmarceau) [inline packages/atomic/src/components/search/result-template-components/atomic-result-link/atomic-result-link.tsx:59]:
```suggestion
   * ```html
   * <atomic-result-link href-template='${clickUri}?id=${raw.itemtitle}'></atomic-result-link>
   * ```
```

**Review by olamothe — APPROVED** [review]

**Comment by jpmarceau** [comment]
Anything I can do to help with the failing checks?

**Pull request merged**